onnx/defs/quantization/defs.cc

```diff
@@ -9,11 +9,12 @@ namespace ONNX_NAMESPACE {
 
 static const char* QuantizeLinear_ver21_doc = R"DOC(
 The linear quantization operator. It consumes a high precision tensor, a scale, and a zero point to compute the low precision / quantized tensor.
-The scale factor and zero point must have same shape, and can be either a scalar for per-tensor / per layer quantization, or a 1-D tensor for per-axis quantization.
+The scale factor and zero point must have same shape, and can be either a scalar for per-tensor / per layer quantization, a 1-D tensor for per-axis quantization,
+or have identical rank to the input for blocked quantization.
 The quantization formula is `y = saturate ((x / y_scale) + y_zero_point)`.
 For saturation, it saturates to [0, 255] if it's uint8, [-128, 127] if it's int8, [0, 65535] if it's uint16, or [-32768, 32767] if it's int16.
 For (x / y_scale), it's rounding to the nearest even. Refer to https://en.wikipedia.org/wiki/Rounding for details.
```

Comment:
```suggestion
The quantization formula is `y = saturate ((x / y_scale) + y_zero_point)`.
For saturation, it saturates to `[0, 255]` if it's `uint8`, `[-128, 127]` if it's `int8`, `[0, 65535]` if it's `uint16`, or `[-32768, 32767]` if it's `int16`.
For `(x / y_scale)`, it's rounding to the nearest even. Refer to https://en.wikipedia.org/wiki/Rounding for details.
```

nit: formatting: format code entities, types etc. as code

Reply:
Fixed as part of rewriting of the quantization operator definition.